product_docs/docs/biganimal/release/free_trial/detail/experiment/backup_and_restore.mdx

```diff
@@ -20,7 +20,7 @@ Now you can switch to your new (and empty) baseball database. You're prompted fo
 \c baseball
 ```
 
-Normally we'd use log files to determine the moment in time that we want to restore. The free trial doesn't provide access to the logs, however. So that we have accurate timing of our activities, let's change the prompt to include a timestamp. For convenience, we'll use the (almost) ISO-8601 format that the restore command accepts.
+Normally you use the log files to determine the moment in time that you want to restore. For the purposes of this demonstration, let's change the prompt to include a timestamp to have an accurate timing of our activities. For convenience, we'll use the (almost) ISO-8601 format that the restore command accepts.
```

Comment:
Should we update the example to use logs instead of a timestamp?

Reply:
why logs? i might be missing some context

Reply:
Not sure I follow, either.  @josh-heyer , would you be able to provide a recommendation here?

Reply:
After discussing with Jason, we determined the answer to my question is no.